README.md

```diff
@@ -23,25 +23,28 @@ sccache is also available as a [GitHub Actions](https://github.com/marketplace/a
 Table of Contents (ToC)
 =======================
 
-* [Installation](#installation)
-* [Usage](#usage)
-* [Build Requirements](#build-requirements)
-* [Build](#build)
-* [Separating caches between invocations](#separating-caches-between-invocations)
-* [Overwriting the cache](#overwriting-the-cache)
-* [Debugging](#debugging)
-* [Interaction with GNU `make` jobserver](#interaction-with-gnu-make-jobserver)
-* [Known Caveats](#known-caveats)
-* [Storage Options](#storage-options)
-  * [Local](docs/Local.md)
-  * [S3](docs/S3.md)
-  * [R2](docs/S3.md#R2)
-  * [Redis](docs/Redis.md)
-  * [Memcached](docs/Memcached.md)
-  * [Google Cloud Storage](docs/Gcs.md)
-  * [Azure](docs/Azure.md)
-  * [GitHub Actions](docs/GHA.md)
-  * [WebDAV (Ccache/Bazel/Gradle compatible)](docs/Webdav.md)
+- [sccache - Shared Compilation Cache](#sccache---shared-compilation-cache)
+- [Table of Contents (ToC)](#table-of-contents-toc)
```

Comment:
It's better not to introduce unrelated changes. Welcome to submit them as seperate PR.